AirCasting/Map/AirMapView.swift

```diff
@@ -66,15 +88,44 @@ struct AirMapView: View {
-                                      placePickerIsUpdating: Binding.constant(false),
-                                      isUserInteracting: $isUserInteracting,
-                                      isSessionActive: session.isActive,
-                                      isSessionFixed: session.isFixed,
-                                      noteMarketTapped: $noteMarkerTapped,
-                                      noteNumber: $noteNumber,
-                                      mapNotes: $mapNotesVM.notes)
+                        switch mapSessionType {
+                        case .active:
+                            // - custom dot
+                            // - draw path when ongoing session updates
+                            // - live tracking
+                            _MapView(path: pathPoints,
+                                     type: .normal,
+                                     trackingStyle: .latestPathPoint,
+                                     userIndicatorStyle: .custom(color: _MapViewThresholdFormatter.shared.color(points: pathPoints, threshold: threshold)),
+                                     locationTracker: MapLocationTrackerAdapter(locationTracker),
+                                     markers: mapNotesVM.notes.asMapMarkers(with: didTapNote))
+                            .addingOverlay { mapView in overlayHeatMap(on: mapView, threshold: threshold) }
+                        case .fixed:
+                            // - custom dot
+                            // - no path drawing
+                            _MapView(path: pathPoints,
+                                     type: .normal,
+                                     trackingStyle: .latestPathPoint,
+                                     userIndicatorStyle: .custom(color: _MapViewThresholdFormatter.shared.color(points: pathPoints, threshold: threshold)),
+                                     locationTracker: ConstantTracker(location: pathPoints.last?.location ?? .applePark))
+                        case .other:
+                            // here only mobileDormant type should be considered
+                            // - custom dot
+                            // - draw already known and finished path
+                            // - no need for real time updated
+                            _MapView(path: pathPoints,
+                                     type: .normal,
+                                     trackingStyle: .wholePath,
+                                     userIndicatorStyle: .none,
+                                     locationTracker: MapLocationTrackerAdapter(locationTracker),
+                                     markers: mapNotesVM.notes.asMapMarkers(with: didTapNote))
+                            .addingOverlay { mapView in overlayHeatMap(on: mapView, threshold: threshold) }
+                        }
+                        #warning("TODO: Implement calculating stats only for visible path points")
```

Comment:
This can be removed. Michael said it is not needed anymore (and I think this comment is not present on develop branch anymore either)